scripts/migration/lib.sh

```diff
@@ -226,7 +226,20 @@ check_version() {
   if [ "${VERSION["${1}-config"]}" = "any" ]; then
     log_warn "skipping version validation of ${1}-config for version \"${VERSION["${1}-config"]}\""
     return
-  elif [[ ! "${VERSION["${1}-config"]}" =~ v[0-9]+\.[0-9]+\.[0-9]+ ]]; then
+  fi
+
+  common_override=$(yq4 '.global.ck8sVersion' "${CK8S_CONFIG_PATH}/common-config.yaml")
+  sc_wc_override=$(yq4 '.global.ck8sVersion' "${CK8S_CONFIG_PATH}/${1}-config.yaml")
+  if [ "$common_override" != "null" ] || [ "$sc_wc_override" != "null" ]; then
+    log_warn "You have set the ck8sVersion in a override config if you continue upgrade $version apply will fail you will have to do it manually if you want to use the override version."
```

Comment:
```suggestion
    log_warn "You have set the ck8sVersion in an override config"
    log_warn "If this override version does not match the current repository version then:"
    log_warn "- upgrade ${version} prepare will not remove it, and 
    log_warn "- upgrade ${version} apply will fail"
```
Just to make it all a bit clearer.